wgpu-hal/src/gles/device.rs

```diff
@@ -237,12 +239,32 @@ impl super::Device {
             binding_array: BoundsCheckPolicy::Unchecked,
         };
 
+        // Update naga options if mesh consist of point list and it is vertex shader
+        let maybe_update_naga_options = if Some(wgt::PrimitiveTopology::PointList)
+            == primitive_topology
+            && naga_stage == naga::ShaderStage::Vertex
```

Comment:
Let's move the conditional to it's own boolean variable called `update_naga_options` and the actual options to just `naga_options`